File: app/src/main/java/myarena/api/API_console.java
package myarena.api;

/**
 * Ответ MyArena.ru API на запрос о консоли сервера (Неофициальный метод - getconsole)
 */
public class API_console extends  API_ответ {

    public String console_log;

    @Override
    public String toString() {
        if (успех()) {
            return console_log;
        }
        else {
            return message;
        }
    }
}

File: app/src/main/java/myarena/api/API_console.kt
package myarena.api

import com.google.gson.GsonBuilder

/**
 * Ответ MyArena.ru API на запрос о консоли сервера (Неофициальный метод - getconsole)
 */
class API_console : API_ответ() {

    private val console_log = "ERROR: Empty output"

    override fun toString(): String {
        return if (успех()) {
            console_log
        } else {
            message
        }
    }

    companion object {
        fun fromJSON(JSON: String): API_console {
            val builder = GsonBuilder()
            val gson = builder.create()
            return gson.fromJson(JSON, API_console::class.java)
        }
    }
}
